### ISSUE TITLE
Git bash terminals in editor jam each other display contents

### ISSUE THREAD

**Issue by julienbr67** [microsoft/vscode#139677]
Issue Type: <b>Bug</b>

If I set git bash as default terminal with default location "editor" and open multiple terminal tabs, the content from the latest opened terminal shows on all the other terminals instead of their own content.

VS Code version: Code 1.63.2 (899d46d82c4c95423fb7e10e68eba52050e30ba3, 2021-12-15T09:40:02.816Z)
OS version: Windows_NT x64 10.0.19043
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz (12 x 2592)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|31.74GB (21.90GB free)|
|Process Argv|--crash-reporter-id 305705ef-8f1b-4259-a0ed-ce6316eb4119|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (12)</summary>

Extension|Author (truncated)|Version
---|---|---
gitlens|eam|11.7.0
themeswitch|Foo|1.0.5
vscode-pull-request-github|Git|0.34.2
python|ms-|2021.12.1559732655
vscode-pylance|ms-|2021.12.2
jupyter|ms-|2021.11.1001550889
jupyter-keymap|ms-|1.0.0
jupyter-renderers|ms-|1.0.4
cpptools|ms-|1.7.1
material-icon-theme|PKi|4.11.0
pdf|tom|1.2.0
markdown-all-in-one|yzh|3.4.0

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythontb:30283811
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscorecescf:30384386
pythondataviewer:30285071
vscod805:30301674
pythonvspyt200:30340761
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593cf:30376535
pythonvs932:30410667
vscop804:30404766
vs360:30404995
vsrem710cf:30416617
vscexrecpromp3t1:30407762
vscop841cf:30415854

```

</details>

**Comment by hilaolu** [comment]
Same problem.

**Comment by atg** [comment]
Regression of #130110

**Comment by meganrogge** [comment]
fixed in insider's w same root cause as #139222 /duplicate
